examples/pulse_control/5_Build_single_qubit_gates.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
<p>1) constructed a</p><p> </p><p>2) This is mainly due to the <strong>simplicity of the</strong>&nbsp;π/2 pulse characterization.</p><p> </p><p>3) we <strong>can</strong> switch to <strong>a</strong> Drag... that will optimize the detuning, phase, length of the pulse, and limit ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>